utils/build/rollup.config.js

```diff
@@ -62,44 +62,31 @@ function header() {
 
 const builds = [
 	{
-		input: 'src/Three.js',
+		input: {
+			'three.module.js': 'src/Three.js',
+			'three.webgpu.js': 'src/Three.WebGPU.js',
+			'three.webgpu.nodes.js': 'src/Three.WebGPU.js',
```

Comment:
This is unused, and I can no longer find references to `three/nodes`. CI still depends on it for size checking.